pkg/detectors/azurecontainerregistry/azurecontainerregistry.go

```diff
@@ -0,0 +1,97 @@
+package azurecontainerregistry
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"encoding/base64"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+    url = regexp.MustCompile(`([a-zA-Z0-9-]{1,100})\.azurecr\.io`)
+    password = regexp.MustCompile(`[A-Za-z0-9+/=]{52}`)
```

Comment:
This needs boundary characters or this part can get a lot of matches.

Reply:
Also looks like you need to run `go fmt -w`. Your editor should do this for you.